src/components/Wishlisted-Product/WishlistedProducts.jsx

```diff
@@ -30,7 +30,7 @@ const WishlistProduct = () => {
                                     {!wishlistedProduct.isInStock ? <div className="badge-type3">Stock Out</div> : ""}
                                     <i className="ap_wishlisted-product-icon"><FaRegHeart /></i>
 
-                                    <div className="card-w-badge-subcontainer1" style= {{ backgroundImage: `url(${ wishlistedProduct.img })`}}>
+                                    <div className="card-w-badge-subcontainer1" style= {{ backgroundImage: `url(${ wishlistedProduct.img[0]})`, backgroundPosition: "center" }}>
```

Comment:
use css classes avoid inline styling as much as you can

Reply:
actually thought that these styling won't be changed anyways in future code, therfore thought of declaring it here itself!!